docs/snippets/vue/list-story-template.vue3.js.mdx

```diff
@@ -0,0 +1,37 @@
+```js
+// List.stories.js
+
+import List from './List.vue';
+import ListItem from './ListItem.vue';
+
+//👇 Imports a specific story from ListItem stories
+import { Unchecked } from './ListItem.stories';
+
+const ListTemplate = (args) => ({
+  components: { List, ListItem },
+  setup() {
+    return { args };
+  },
+  template: `
+    <List v-bind="args">
+        <div v-for="item in items" :key="item.someString">
```

Comment:
Does `setup()` need to include `items` in the returned object to have access to it here?

Reply:
This is one is similar to the Table example. I've updated it with a recent commit.